### ISSUE TITLE
(Docker) PermissionError: [Errno 13]

### ISSUE THREAD

**Issue by joaopcanario** [datasciencebr/serenata-de-amor#276]
I don't know what happened but after last updates from master branch I'm seeing this error on logs of my container:

`PermissionError: [Errno 13] Permission denied: '/Home/jovyan/.jupyter/migrated'`

I started to noticed as well that Jupyter isn't saving the .html and .py files of my notebook anymore. Someone already saw this?

**Comment by joaopcanario** [comment]
And here my container log:

```
[I 12:51:12.660 NotebookApp] Saving file at /work/develop/2017-08-20-joaopcanario-companies-analisys.ipynb
[W 12:51:12.661 NotebookApp] Saving untrusted notebook work/develop/2017-08-20-joaopcanario-companies-analisys.ipynb
Traceback (most recent call last):
  File "/opt/conda/bin/jupyter-nbconvert", line 6, in <module>
    sys.exit(nbconvert.nbconvertapp.main())
  File "/opt/conda/lib/python3.6/site-packages/jupyter_core/application.py", line 267, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "/opt/conda/lib/python3.6/site-packages/traitlets/config/application.py", line 657, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-7>", line 2, in initialize
  File "/opt/conda/lib/python3.6/site-packages/traitlets/config/application.py", line 87, in catch_config_error
    return method(app, *args, **kwargs)
  File "/opt/conda/lib/python3.6/site-packages/nbconvert/nbconvertapp.py", line 257, in initialize
    super(NbConvertApp, self).initialize(argv)
  File "<decorator-gen-6>", line 2, in initialize
  File "/opt/conda/lib/python3.6/site-packages/traitlets/config/application.py", line 87, in catch_config_error
    return method(app, *args, **kwargs)
  File "/opt/conda/lib/python3.6/site-packages/jupyter_core/application.py", line 243, in initialize
    self.migrate_config()
  File "/opt/conda/lib/python3.6/site-packages/jupyter_core/application.py", line 169, in migrate_config
    migrate()
  File "/opt/conda/lib/python3.6/site-packages/jupyter_core/migrate.py", line 241, in migrate
    with open(os.path.join(env['jupyter_config'], 'migrated'), 'w') as f:
PermissionError: [Errno 13] Permission denied: '/home/jovyan/.jupyter/migrated'
[E 12:51:13.673 NotebookApp] Post-save hook failed o-n /home/jovyan/work/develop/2017-08-20-joaopcanario-companies-analisys.ipynb
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.6/site-packages/notebook/services/contents/filemanager.py", line 129, in run_post_save_hook
        self.post_save_hook(os_path=os_path, model=model, contents_manager=self)
      File "/home/jovyan/.jupyter/jupyter_notebook_config.py", line 49, in post_save
        check_call(['jupyter', 'nbconvert', '--to', 'script', fname], cwd=d)
      File "/opt/conda/lib/python3.6/subprocess.py", line 291, in check_call
        raise CalledProcessError(retcode, cmd)
    subprocess.CalledProcessError: Command '['jupyter', 'nbconvert', '--to', 'script', '2017-08-20-joaopcanario-companies-analisys.ipynb']' returned non-zero exit status 1.
[W 12:51:13.675 NotebookApp] Unexpected error while running post hook save: Command '['jupyter', 'nbconvert', '--to', 'script', '2017-08-20-joaopcanario-companies-analisys.ipynb']' returned non-zero exit status 1.
[E 12:51:13.676 NotebookApp] {
      "Host": "localhost:8888",
      "Connection": "keep-alive",
      "Content-Length": "99047",
      "Accept": "application/json, text/javascript, */*; q=0.01",
      "Origin": "http://localhost:8888",
      "X-Requested-With": "XMLHttpRequest",
      "X-Xsrftoken": "2|f3d119ef|18ce86aec2cf0a91ceaddd740ca37452|1502814126",
      "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.101 Safari/537.36",
      "Content-Type": "application/json",
      "Dnt": "1",
      "Referer": "http://localhost:8888/notebooks/work/develop/2017-08-20-joaopcanario-companies-analisys.ipynb",
      "Accept-Encoding": "gzip, deflate, br",
      "Accept-Language": "en-US,en;q=0.8",
      "Cookie": "_xsrf=2|f3d119ef|18ce86aec2cf0a91ceaddd740ca37452|1502814126"
    }
[E 12:51:13.676 NotebookApp] 500 PUT /api/contents/work/develop/2017-08-20-joaopcanario-companies-analisys.ipynb (172.19.0.1) 1017.55ms referer=http://localhost:8888/notebooks/work/develop/2017-08-20-joaopcanario-companies-analisys.ipynb
```

**Comment by joaopcanario** [comment]
Checking a little more about the last update, I found that changing back the tag of `jupyter/datascience-notebook` to `bb222f49222e` fix this problem. I'll try track watch changed in this image to found a better solution.

Besides that, googling a little about this error, I found this [link](https://blog.openshift.com/jupyter-on-openshift-part-6-running-as-an-assigned-user-id/) and I guess that may help to found a solution as well.
